src/WebJobs.Extensions.DocumentDB/Trigger/DocumentDBTriggerConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
+{
+    internal static class DocumentDBTriggerConstants
+    {
+        public const string DefaultLeaseCollectionName = "leases";
+
+        public const string TriggerName = "DocumentDBTrigger";
+
+        public const string TriggerDescription = "New changes on collection {0} at {1}";
```

Comment:
I tried to implement it, but I don't seem to be able to read the data within the `ToParameterDescriptor` implementation.